web/modules/custom/weather/src/Form/WeatherUserForm.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Drupal\weather\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\weather\Services\WeatherApi;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * {@inheritdoc}
+ */
+class WeatherUserForm extends FormBase {
+
+  /**
+   * Constructor.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory,
```

Comment:
You need to write a little more than just "Constructor".
Example: Constructs a new WeatherUserForm